src/main/kotlin/no/nav/amt/distribusjon/varsel/VarselProducer.kt

```diff
@@ -0,0 +1,49 @@
+package no.nav.amt.distribusjon.varsel
+
+import no.nav.amt.distribusjon.Environment
+import no.nav.amt.distribusjon.kafka.Producer
+import no.nav.amt.distribusjon.kafka.config.KafkaConfig
+import no.nav.amt.distribusjon.kafka.config.KafkaConfigImpl
+import no.nav.amt.distribusjon.kafka.config.LocalKafkaConfig
+import no.nav.amt.distribusjon.varsel.model.Varsel
+
+class VarselProducer(
+    kafkaConfig: KafkaConfig = if (Environment.isLocal()) LocalKafkaConfig() else KafkaConfigImpl(),
+) {
+    private val producer = Producer(kafkaConfig, Environment.MINSIDE_VARSEL_TOPIC)
+
+    private val skalTogglesAv = !Environment.isLocal()
```

Comment:
Skal den ikke kjøre i dev? Og, hvis dette er snakk om featuretoggling, kanskje vi heller bør bruke unleash? Mulig jeg misforstår 😅

Reply:
Jeg føler ikke at vi er helt klare for å begynne å produsere varsler i dev helt enda, selv om det kanskje ikke skader.
Siden tanken er at dette er en midlertidig toggle årker jeg ikke sette opp unleash for det 🙂